src/v2/view-builder/components/AuthenticatorEnrollOptions.js

```diff
@@ -16,19 +16,21 @@ import hbs from 'handlebars-inline-precompile';
 
 const AuthenticatorRow = View.extend({
   className: 'authenticator-row clearfix',
-  template: hbs('\
-        <div class="authenticator-icon-container">\
-          <div class="factor-icon authenticator-icon {{iconClassName}}">\
-          </div>\
-        </div>\
-        <div class="authenticator-description">\
-          <h3 class="authenticator-label no-translate">{{label}}</h3>\
-          {{#if description}}\
-            <p class="authenticator-description--text">{{description}}</p>\
-          {{/if}}\
-          <div class="authenticator-button" {{#if buttonDataSeAttr}}data-se="{{buttonDataSeAttr}}"{{/if}}></div>\
-        </div>\
-      '),
+  template: hbs`
+    <div class="authenticator-icon-container">
+      <div class="factor-icon authenticator-icon {{iconClassName}}"></div>
+    </div>
+    <div class="authenticator-description">
+      <h3 class="authenticator-label no-translate">{{label}}</h3>
+      {{#if description}}
+        <p class="authenticator-description--text">{{description}}</p>
+      {{/if}}
+      {{#if authenticatorUsageText}}
+        <p class="authenticator-usage-text">{{authenticatorUsageText}}</p>
```

Comment:
we can use the existing courage `explain` class ?

Reply:
the text requires specific color and other css props. adding `explain` or `explain-text` does not add any css values.

Reply:
would it make sense now to use inbuilt class ? since now we don't have any other values.

Reply:
we are still passing the color `#a7a7a7`. could you be specific on what inbuilt class you are referring to?

Reply:
The className was not top of my ahead. But here you go, i search for it. 

https://github.com/okta/okta-signin-widget/blob/master/assets/sass/okta-theme.scss#L54

Reply:
`$medium-text-color` is set to `#777` but UX wants `#a7a7a7`